src/TemplateData.js

```diff
@@ -338,8 +338,13 @@ class TemplateData {
   }
 
   static cleanupData(data) {
-    if ("tags" in data && typeof data.tags === "string") {
-      data.tags = data.tags ? [data.tags] : [];
+    if ("tags" in data){
+      if (typeof data.tags === "string"){
+        data.tags = data.tags ? [data.tags] : [];
+      }
+      if (data.tags === null){
```

Comment:
This should be an `else if` since the conditions are mutually exclusive. When `typeof data.tags === "string"`, the second condition is evaluated again even though it cannot be true.